crest/Assets/Crest/Crest/Scripts/Shapes/ShapeGerstner.cs

```diff
@@ -262,10 +262,15 @@ public void CrestUpdate(CommandBuffer buf)
 #if UNITY_EDITOR
         void UpdateEditorOnly()
         {
-            if (_spectrum == null)
+            if (_spectrum != null)
+            {
+                _activeSpectrum = _spectrum;
+            }
+            // TODO - dont do this. could init to default spectrum asset in Reset() function
```

Comment:
Is this something that should be resolved before merging?

Reply:
yeah i think we need to kill that. we have a "Create" button now to create spectrum assets. so allowing to edit a dynamically created pseudo-asset is evil. do you want to do the honours or shall i?

Reply:
I don't think users can edit the dynamic one since _activeSpectrum is not serialised. So you are planning on removing the dynamic spectrum completely? Honours is yours :)

Reply:
Hm i've lost a handle on what i had in mind here. I think i found the dynamically created settings were editable, but as you pointed out .. they're not :). I tested now and the behaviour is fine so lets nuke this todo. ill do this now.